api-contract/contract.yaml

```diff
@@ -276,6 +416,19 @@ components:
           type: integer
         posts:
           type: integer
+    Posts:
+      type: object
+      properties:
+        post:
+          type: string
+        canLike:
+          type: boolean
+        canUnlike:
+          type: boolean
+        likes:
```

Comment:
I would probably rename to 'likeCount', similarly I would probably add another one for the comments, 'commentCount'.

What do you think if we add another property for when the post was created? Something like createdAt